### PR TITLE
qa: use correct imports to resolve fuse_mount and kernel_mount

### PR DESCRIPTION
**Pull request by mchangir**
Fixes: https://tracker.ceph.com/issues/62706

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: qa/tasks/cephfs/test_snap_schedules.py [hunk 1]
```diff
@@ -588,8 +588,8 @@ def remove_snapshots(self, dir_path, sdn):
             self.mount_a.run_shell(['rmdir', snapshot_path])
 
     def get_snap_dir_name(self):
-        from tasks.cephfs.fuse_mount import FuseMount
-        from tasks.cephfs.kernel_mount import KernelMount
+        from .fuse_mount import FuseMount
```
Comment (vshankar) [inline qa/tasks/cephfs/test_snap_schedules.py:591]: How come this fails only intermittently then?

Comment (mchangir) [reply qa/tasks/cephfs/test_snap_schedules.py:591]: this, I have no idea about
I humbly request Python experts to lend a hand here

**Comment by vshankar** [comment]
jenkins test api

File: qa/tasks/cephfs/test_snap_schedules.py (hunk 1 above)
Comment (vshankar) [reply qa/tasks/cephfs/test_snap_schedules.py:591]: Its likely that something is messing up PYTHONPATH causing this to error out.

**Comment by vshankar** [comment]
@mchangir Does the teuthology job passes with this change? I don't mind merging this till the actual pythonpath thing is resolved.

**Comment by mchangir** [comment]
> @mchangir Does the teuthology job passes with this change? I don't mind merging this till the actual pythonpath thing is resolved.

FYI - https://pulpito.ceph.com/mchangir-2023-11-15_07:18:03-fs-main-distro-default-smithi/

**Review by vshankar — APPROVED** [review]

**Pull request merged**